jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt

```diff
@@ -259,7 +260,8 @@ class CodeWhispererService {
 
                 if (
                     CodeWhispererConstants.Customization.customizationNotFoundExceptionPredicate(e) ||
```

Comment:
what does customizationNotFoundExceptionPredicate check for?

Reply:
```
e is AccessDeniedException && (e.message?.contains(invalidCustomizationMessage, ignoreCase = true) ?: false)
```

ADE & e.messagecontains("not found")

Reply:
do we need this or we can just update this one to check for RNF?

Reply:
yea let me remove it